docs/guides/admin/docs/configuration/basic.md

```diff
@@ -1,108 +1,114 @@
+    * The user for the communication between Opencast nodes. It is sometimes also used by capture agents.
+      This is set to `opencast_system_account` by default.
 * `org.opencastproject.security.digest.pass`
-    * The password for the communication between Opencast nodes and capture agents. This is set to `CHANGE_ME` by default.
-
-*Note:* The digest credentials are also used for internal communication of Opencast servers. So these keys have to be
-set to the same value on each of your Opencast nodes (Core, Worker, Capture Agent, …)
+    * The password for the communication between Opencast nodes. It is sometimes also used by capture agents.
+      This is set to `CHANGE_ME` by default.
+* `karaf.shutdown.command`
+    * The security token used for shutting down Opencast. Set this to a random string.
 
+Make sure that these settings are identical on all nodes of the cluster.
 
-Step 3: Change the default shutdown command
--------------------------------------------
 
-Karaf provides a socket over which you can send a shutdown command. The socket does not provide any kind of
-authentication. Therefore anyone who obtains write access to this socket is able to shutdown karaf and everything
-that runs on it. There is a default `karaf.shutdown.command` defined in `custom.properties`. Change this to something
-secret.
+Step 3: Setting up Apache ActiveMQ Message Broker
+-------------------------------------------------
 
+Opencast requires Apache ActiveMQ to relay messages between micro-services.
+For configuration details, please follow the:
 
-Step 4: Setting up Apache ActiveMQ Message Broker
--------------------------------------------------
-<!-- _This is a very nonchalant and subdued description for how important this config probably is -->
-Since version 2.0, Opencast requires a running Apache ActiveMQ instance with a specific configuration.  The message
-broker is mostly run on the admin server of Opencast but can be run separately. It needs to be started before Opencast.
-For more details about the setup, have a look at the [Apache ActiveMQ configuration guide](message-broker.md).
+- [Apache ActiveMQ configuration guide](message-broker.md)
 
 
-Step 5: Database Configuration
+Step 4: Database Configuration
 ------------------------------
 
-Opencast uses an integrated H2 database by default. While you will find it perfectly functional, it has certain
-drawbacks:
+Opencast uses an integrated H2 database by default, which has certain drawbacks:
 
-* It is rather slow
 * It cannot be used for distributed set-ups
 * Upgrading Opencast with this database is not possible
 
-For testing, it is totally fine to keep the internal database, but you are highly encouraged to switch to a stand-alone
-database for productional use. For more information about database configuration, have a look at the [Database
-Configuration](database.md) section.
+For testing, it is fine to keep the internal database, but configure a stand-alone database for production.
```

Comment:
`For testing, it is fine to keep the internal database` -> `For testing, the internal database will suffice,`

`but configure a stand-alone database for production.` -> `but for production, a stand-alone database must/should be configured.`

Reply:
Hah, so I thought about commenting on this, but decided against since it's not *technically* wrong.  I would rewrite this as
```The internal database will suffice for testing, however for production uses a stand-alone database is required```

Optionally we could include a list of databases, but I would expect anyone reading this to know what their options and preferences are.

Reply:
You're right, it's not technically wrong x) but the OG syntax muddles the message a bit, hence the comment.

How about
`The internal database will suffice for testing, however a stand-alone database is required for production uses.`
?

Reply:
That's a cleaner way of saying it, +1

Reply:
Went with

> The internal database will suffice for testing, however a stand-alone database is required for production uses.